Dockerfile.frontend

```diff
@@ -1,7 +1,10 @@
-FROM geometalab/python3-gis
+FROM geometalab/gdal-debian
+
+USER root
```

Comment:
Should we switch back to user `py` at the end (or whereever the Dockerfile doesn't need `root` anymore) or does our entrypoint need `root`, too?